### PR TITLE
Release notes for 6.8.13

### PR DESCRIPTION
**Pull request by pgomulka**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS and architecture that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

File: docs/reference/release-notes/6.8.asciidoc [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+coming[6.8.13]
+
+Also see <<breaking-changes-6.8,Breaking changes in 6.8>>.
+
+[[enhancement-6.8.13]]
+[float]
+=== Enhancements
+
+Features/Java Low Level REST Client::
+* [client] Add simple support for gzip compression {es-pull}63230[#63230] (issue: {es-issue}62044[#62044])
```
Comment (jrodewig) [inline docs/reference/release-notes/6.8.asciidoc:14]:
```suggestion
* Add simple support for gzip compression {es-pull}63230[#63230] (issue: {es-issue}62044[#62044])
```

File: docs/reference/release-notes/6.8.asciidoc [hunk 2]
```diff
@@ -1,3 +1,54 @@
...
+
+CCR::
+* CCR should retry on CircuitBreakingException {es-pull}62013[#62013] (issue: {es-issue}55633[#55633])
+* Retry CCR shard follow task when no seed node left {es-pull}63225[#63225]
+* Set timeout of auto put-follow request to unbounded {es-pull}61679[#61679] (issue: {es-issue}56891[#56891])
+
+Features/Java Low Level REST Client::
+* [client] Decompressed gzip responses report inconsistent encoding values [ISSUE] {es-pull}63314[#63314]
+
+MULTIPLE AREA LABELS::
```
Comment (jrodewig) [inline docs/reference/release-notes/6.8.asciidoc:37]:
```suggestion
Features/Java High Level REST Client::
```

File: docs/reference/release-notes/6.8.asciidoc [hunk 3]
```diff
@@ -1,3 +1,54 @@
...
+* Ensure authz operation overrides transient authz headers {es-pull}61621[#61621]
+* Relax the index access control check for scroll searches {es-pull}61446[#61446]
+
+CCR::
+* CCR should retry on CircuitBreakingException {es-pull}62013[#62013] (issue: {es-issue}55633[#55633])
+* Retry CCR shard follow task when no seed node left {es-pull}63225[#63225]
+* Set timeout of auto put-follow request to unbounded {es-pull}61679[#61679] (issue: {es-issue}56891[#56891])
+
+Features/Java Low Level REST Client::
+* [client] Decompressed gzip responses report inconsistent encoding values [ISSUE] {es-pull}63314[#63314]
```
Comment (jrodewig) [inline docs/reference/release-notes/6.8.asciidoc:35]: This looks like an issue, not a PR.
```suggestion
```

File: docs/reference/release-notes/6.8.asciidoc [hunk 4]
```diff
@@ -1,3 +1,54 @@
...
+
+MULTIPLE AREA LABELS::
+* Add hlrc_rest_total_hits_as_int parameter in scroll request {es-pull}61685[#61685] (issue: {es-issue}61677[#61677])
+* Fix bwc of HLRC scroll search requests {es-pull}61696[#61696] (issue: {es-issue}61677[#61677])
```
Comment (jrodewig) [inline docs/reference/release-notes/6.8.asciidoc:39]: This PR was closed but not merged.
```suggestion
```

**Review by jrodewig — APPROVED** [review]
LGTM. I left a few suggestions that clean up the labels and remove unmerged PRs. Thanks @pgomulka!

**Comment by lcawl** [comment]
We don't have any ML PRs to add, so this LGTM too

File: docs/reference/release-notes/6.8.asciidoc (hunk 4 above)
Comment (pgomulka) [reply docs/reference/release-notes/6.8.asciidoc:39]: agree - it was closed in favour of 61685 (already mentioned)

**Comment by pgomulka** [comment]
@elasticmachine run elasticsearch-ci/docs-check

**Comment by pgomulka** [comment]
@elasticmachine run elasticsearch-ci/oss-distro-docs

**Comment by pgomulka** [comment]
@elasticmachine run elasticsearch-ci/oss-distro-docs

**Comment by pgomulka** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by pgomulka** [comment]
@jrodewig would you be able to give it a quick second look? I have added release highlights

File: docs/reference/release-notes/highlights-6.8.13.asciidoc [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+<titleabbrev>6.8.13</titleabbrev>
+++++
+
+See also <<release-notes-6.8.13,{es} 6.8.13 release notes>>.
+
+[float]
+==== Fixed a retry logic for {ccr} when encountering exceptions
+
+A {ccr} will automatically retry when facing transient errors caused by circuit breaker or
+inability to connect to remote cluster.
```
Comment (jrodewig) [inline docs/reference/release-notes/highlights-6.8.13.asciidoc:13]:
```suggestion
==== Fixed retries for {ccr}

{ccr-cap} now retries operations that failed due to a circuit breaker or a lost
remote cluster connection.
```

**Review by jrodewig — APPROVED** [review]
LGTM. I updated the text for the release highlight with 553358b34065fd012aa91b5be4e9901001824d50. Thanks!

**Comment by jrodewig** [comment]
Going to merge this so it's ready for the release.

**Comment by pgomulka** [comment]
thank you @jrodewig !!

**Pull request merged**